packages/Pager/src/pager.scss

```diff
@@ -1,192 +1,292 @@
+    --ids-pager-item-color: #{tokens.$grey-800};
+    --ids-pager-item-radius: 0;
+    --ids-pager-item-min-width: 3.2rem;
+    --ids-pager-item-z-index: #{tokens.$z-index-10};
+    --ids-pager-item-vertical-offset: -.1rem;
+    --ids-pager-item-prev-next-offset: -.1rem;
+
+    /* Selected Item */
+    --ids-pager-item-selected-background: #{tokens.$electric-blue-500};
+    --ids-pager-item-selected-color: #{tokens.$samoyed};
+
+    /* Hover */
+    --ids-pager-item-hover-border: .1rem solid #{tokens.$grey-600};
+    --ids-pager-item-hover-background: transparent;
+
+    /* Focus */
+    --ids-pager-item-focus-border-width: .1rem;
+    --ids-pager-item-focus-outline-width: 0;
+    --ids-pager-item-focus-color: #{tokens.$electric-blue-600};
+
+    /* Disabled */
+    --ids-pager-item-disabled-background: #{tokens.$grey-300};
+    --ids-pager-item-disabled-border: .1rem solid #{tokens.$grey-300};
+    --ids-pager-item-disabled-color: #{tokens.$grey-600};
+
+    /* Arrows */
+    --ids-pager-arrow-color: #{tokens.$grey-500};
+
+    /* Ellipsis */
+    --ids-pager-ellipsis-border: .1rem solid transparent;
+    --ids-pager-ellipsis-color: #{tokens.$grey-600};
+    --ids-pager-ellipsis-offset: 0;
+
+    /* Results */
+    --ids-pager-results-color: #{tokens.$grey-600};
+    --ids-pager-results-margin: 0 0 0 #{tokens.$space-3};
+}
 
-  /* Results */
-  --ids-pager-results-color: #{tokens.$grey-600};
-  --ids-pager-results-margin: 0 0 0 #{tokens.$space-3};
+[data-brand="workleap"] {
+    /* Default */
+    --ids-pager-font-family: var(--hop-body-sm-font-family);
+    --ids-pager-font-size: var(--hop-body-sm-font-size);
+    --ids-pager-height: 2rem;
+    --ids-pager-line-height: var(--hop-body-sm-line-height);
+
+    /* Item */
+    --ids-pager-item-wrapper-border-size: 0.0625rem;
+    --ids-pager-item-wrapper-padding: calc(var(--hop-space-inset-xs) - var(--ids-pager-item-wrapper-border-size));
+    --ids-pager-item-wrapper-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-padding: 0;
+    --ids-pager-item-background: var(--hop-neutral-surface);
+    --ids-pager-item-border: none;
+    --ids-pager-item-prev-border-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-next-border-radius: var(--hop-shape-rounded-md);
+    --ids-pager-item-color: var(--hop-neutral-text);
+    --ids-pager-item-radius: 999px;
+    --ids-pager-item-min-width: 2rem;
+    --ids-pager-item-z-index: 1;
+    --ids-pager-item-vertical-offset: 0;
+    --ids-pager-item-prev-next-offset: var(--hop-space-inline-xs);
+
+    /* Selected Item */
+    --ids-pager-item-selected-background: var(--hop-neutral-surface-strong);
+    --ids-pager-item-selected-color: var(--hop-neutral-text-strong);
+
+    /* Hover */
+    --ids-pager-item-hover-border: none;
+    --ids-pager-item-hover-background: var(--hop-neutral-surface-hover);
+
+    /* Focus */
+    --ids-pager-item-focus-border-width: 0;
+    --ids-pager-item-focus-outline-width: 0.09375rem;
```

Comment:
There is an eslint rule that limits the number after the decimal point, use `0.0938rem`

Reply:
That would not equals 1.5px. Base 16 uses a lot of decimals versus base 10. As in the end screens renders px's I would rethink this rule, what is the goal of that rule?

Reply:
This rule has been defined in the standards of the [wl-web-config](https://github.com/gsoft-inc/wl-web-configs) library, so you can open an issue to have it changed.